integration_tests/Cargo.toml

```diff
@@ -58,6 +58,8 @@ regex = { workspace = true }
 serde = { workspace = true }
 thiserror = { workspace = true }
 libsecp256k1 = { workspace = true }
+export_macro = { path = "./macro" }
+ctor = "0.2.5"
```

Comment:
Is it possible to do some form of macro re-export from the `export_macro` crate? That way, `integration_tests` wouldn't need to explicitly import this. Instead, the `export_test` macro would use `export_macro::ctor` (or something like that).

Reply:
It's a nice idea, unfortunately crates marked proc-macro can't publically export anything other than proc-macros

Reply:
But `ctor` is a proc macro, right?

Reply:
I tried it but the compiler is still giving me hassle. It looks like it won't work unless the macro is defined within this crate itself.

There might be some way to wrap it in something and re-export it but I can't figure it out for now.

Reply:
Eh, fair enough. This isn't a public API so it's not worth the hassle.